itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -589,6 +590,72 @@ public void testIncrementalAdds() throws IOException {
   }
 
   @Test
+  public void testIncrementalLoadWithVariableLengthEventId() throws IOException, TException {
```

Comment:
I don't think, we will increase the number of events in future. But, still I like the idea of setting TRUNCATE event ID as 100 and others starts from 1. Will try and see if any issues with this.